src/org.hdfgroup.hdfview/hdf/view/MetaDataView/DefaultBaseMetaDataView.java

```diff
@@ -54,7 +54,7 @@
 
 import hdf.hdf5lib.H5;
 import hdf.hdf5lib.HDF5Constants;
-import hdf.object.Attribute;
```

Comment:
AttributeDataset is the replacement for Attribute class. Attribute class treated data as strings.
AttributeDataset uses the Dataset class as a base for treating attribute data the same as dataset types.